src/js/modules/Contact/Address.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import MapMarkerSVG from 'js/svg/MapMarker';
+import PropTypes from 'prop-types';
+
+const Address = ({ className, name, location }) => (
+  <div className={`${className} coa-ContactAddress`}>
```

Comment:
The more I look at this, I see the problem in how I structured contact items. Instead of passing a class through (which couples this component to a parent) we should just statically class all of the contact components with 'coa-ContactDetail' or 'coa-ContactItem' or something so the individual contact components can have standalone styles.

Reply:
Not sure what you mean when you say "statically class" but in general this sounds good. I'll add this as technical debt to the backlog.

Reply:
going to give what I think you're saying a shot.